girderformindlogger/api/v1/user.py

```diff
@@ -812,19 +812,21 @@ def getUsersDetails(self):
         .errorResponse(('You do not have write access for this user.',
                         'Must be an admin to create an admin.'), 403)
     )
+
     def updateUser(
         self,
         user,
+        login=None,
         displayName="",
         email="",
         admin=False,
         status=None,
-        firstName=None,
-        lastName=None
```

Comment:
I'm not sure if any frontends are still using the `lastName` parameter of this deprecated call.